.buildkite/pipelines/build_windows.json.py

```diff
@@ -99,6 +101,15 @@ def main(args):
                         choices=actions,
                         default="build",
                         help="Specify a build action.")
+    parser.add_argument("--snapshot",
+                        required=False,
+                        choices=build_snapshot,
+                        default=None,
+                        help="Specify if a snapshot build is wanted.")
+    parser.add_argument("--candidate",
+                        required=False,
+                        default=None,
+                        help="Specify a build candidate string.")
```

Comment:
```suggestion
                        help="Specify a version qualifier.")
```

"Build candidate" is the wrong term. We have build candidates even for versions that have no qualifier.